### DISCUSSION TITLE
Release Schedule (PR 5070)

### DISCUSSION THREAD

**Discussion by haweston** [ajaxorg/ace#5075] [General]
I've got a project that's trying to do a bunch of library upgrades and https://github.com/ajaxorg/ace/pull/5070 looks like it should fixing an issue with where file loader is causing issues SSL issues in Node 18, as we've already switched to webpack5. Is there an ETA on when an official release with that PR would go out?

**Comment by nightwing** [comment]
published 1.15.3 just now
